packages/spor-react/src/linjetag/LineIcon.tsx

```diff
@@ -73,9 +84,16 @@ export const LineIcon = forwardRef<HTMLDivElement, LineIconProps>(
     if (!Icon) {
       return null;
     }
+
+    if (foregroundColor) {
+      styles.iconContainer.backgroundColor = disabled
+        ? "silver"
+        : foregroundColor;
+    }
+
     return (
-      <Box css={styles.iconContainer}>
-        <Icon sx={styles.icon} />
+      <Box css={styles.iconContainer} style={style}>
```

Comment:
litt usikker på hva denne klarer å fange opp med styles. Som igjen betyr at den ikke får stiler fra andre komponenter?

Reply:
Mulig man må skrive det slik
```suggestion
      <Box css={{ ...styles.iconContainer, ...style }}>
```